xml/System.IO.Log/SequenceNumber.xml

```diff
@@ -292,7 +292,7 @@
         <returns>
           <see langword="true" /> if both <see cref="T:System.IO.Log.SequenceNumber" /> instances are not equal, otherwise, <see langword="false" />.</returns>
         <remarks>
-          <format type="text/markdown"><![CDATA[The equivalent method for this operator is <xref:System.IO.Log.SequenceNumber.Compare%28System.IO.Log.SequenceNumber%2CSystem.IO.Log.SequenceNumber%29?displayProperty=fullName>]]></format>
+          <format type="text/markdown"><![CDATA[The equivalent method for this operator is <xref:System.IO.Log.SequenceNumber.CompareTo*></format>
```

Comment:
I think IIRC I was thinking that `CompareTo` returns an `int` and `Equals` returns a `bool`, so I thought that `Equals` would be a better match (and thus equivalent method) to what `op_Inequality` operator returns, which is a `bool`. (I might be a SillyRex:tm: tho. :grinning:)